Refresh.GameServer/Workers/RequestStatisticSubmitWorker.cs

```diff
@@ -0,0 +1,24 @@
+using Bunkum.Core.Storage;
+using NotEnoughLogs;
+using Refresh.GameServer.Database;
+using Refresh.GameServer.Services;
+
+using TrackingService = Refresh.GameServer.Services.RequestStatisticTrackingService;
+
+namespace Refresh.GameServer.Workers;
+
+public class RequestStatisticSubmitWorker : IWorker
+{
+    public int WorkInterval => 5_000;
+    
+    public void DoWork(Logger logger, IDataStore dataStore, GameDatabaseContext database)
+    {
+        lock (TrackingService.TrackerLock)
+        {
+            database.IncrementGameRequests(TrackingService.GameRequestsToSubmit);
+            database.IncrementApiRequests(TrackingService.ApiRequestsToSubmit);
+            
+            TrackingService.ClearRequests();
+        }
```

Comment:
This can be done lockless with [`Increment.Exchange`](https://learn.microsoft.com/en-us/dotnet/api/system.threading.interlocked.exchange?view=net-8.0#system-threading-interlocked-exchange(system-int32@-system-int32))

Just exchange the value with 0, and pass the returned value into the increment calls